core/domain/certificate_assessment_services.py

```
@@ -550,18 +549,42 @@ def start_certificate_assessment_attempt(
+            )
+            remaining_cooldown = cooldown - (
+                datetime.datetime.utcnow() - most_recent_attempt.started_at
             )
+            if remaining_cooldown > datetime.timedelta(seconds=0):
+                remaining_minutes = int(
+                    remaining_cooldown.total_seconds() // 60
+                )
+                if remaining_minutes >= 1:
+                    raise utils.ValidationError(
+                        'You just started an assessment before this time. '
+                        'Please try again in %d minute(s).' % remaining_minutes
```

Ruff (0.16.1)

[warning] Use format specifiers instead of percent format

(UP031)